### PR TITLE
fix(core/ui): don't reparse markdown

### PR DESCRIPTION
**Pull request by marcoscaceres**
The double-parsing can cause the markdown parser to crash.

### CODE REVIEW

File: src/core/ui.js [hunk 1]
```diff
@@ -276,15 +276,14 @@ function rsErrorToHTML(err) {
...
       }</strong> times at:</p>
-    ${err.elements.map(generateMarkdownLink)}
-    `
+    ${markdownToHtml(err.elements.map(generateMarkdownLink).join(""))}`
     : "";
   const details = err.details
     ? `\n\n<details>\n${err.details}\n</details>\n`
     : "";
   const msg = markdownToHtml(`**${err.message}**`, { inline: true });
   const text = `${msg}${hint}${elements}${details}${plugin}`;
```
Comment (sidvishnoi) [inline src/core/ui.js:285]: Let's call this `html` then?

File: src/core/ui.js [hunk 2]
```diff
@@ -276,15 +276,14 @@ function rsErrorToHTML(err) {
     ? `<p class="respec-occurrences">Occurred <strong>${
         err.elements.length
       }</strong> times at:</p>
-    ${err.elements.map(generateMarkdownLink)}
-    `
+    ${markdownToHtml(err.elements.map(generateMarkdownLink).join(""))}`
```
Comment (sidvishnoi) [inline src/core/ui.js:279]: Nit: let's do `join('\n')` instead?

**Comment by sidvishnoi** [comment]
Any example when it crashed?

**Review by sidvishnoi — APPROVED** [review]

File: src/core/ui.js (hunk 1 above)
Comment (marcoscaceres) [reply src/core/ui.js:285]: called it result instead, as we use "html" for actual parsed html.

**Comment by marcoscaceres** [comment]
> Any example when it crashed?

Oh, argh... it was a weird one... I should have kept it.

**Pull request merged**